scripts/deployAll.js

```diff
@@ -0,0 +1,127 @@
+
+const { ethers } = require("hardhat");
+const path = require("path");
+const fs = require("fs");
+
+
+// Arguments required to deploy the TTPSC token contract
+const tokenArgs = {TOKEN_NAME: "TTPSC token", TOKEN_SYMBOL: "TTPSC", INITIAL_SUPPLY: 1000}
+
+/*
+Function that deploys TTPSC token contract
+RETURN: TTPSC token
+*/
+const deployToken = async () => {
+
+    const Token = await ethers.getContractFactory("TTPSC");
+    const ttpscToken = await Token.deploy(
+        tokenArgs.TOKEN_NAME,
+        tokenArgs.TOKEN_SYMBOL,
+        tokenArgs.INITIAL_SUPPLY
+    );
+
+    await ttpscToken.deployed();
+    console.log("TTPSC token deployed at address: ", ttpscToken.address);
+
+    return ttpscToken;
+}
+
+
+/*
+Function that deploys payment manager contract and transfers funds to its address
+PARAM: TTPSC token contract
+RETURN: payment manager contract
+*/
+const deployPaymentsManager = async token => {
+    
+    const PaymentsManager = await ethers.getContractFactory("PaymentsManager");
+    const paymentsManager = await PaymentsManager.deploy(token.address);
+
+    await paymentsManager.deployed();
+    console.log("Payments manager contract deployed at address: ", paymentsManager.address);
+
+    await token.transfer(paymentsManager.address, tokenArgs.INITIAL_SUPPLY);
```

Comment:
Method extracting is not really neccessary at this moment, but add console.log() here

Reply:
Sure!